xbmc/addons/AddonRepos.cpp

```diff
@@ -482,9 +483,10 @@ bool CAddonRepos::FindDependency(const std::string& dependsId,
 
 bool CAddonRepos::FindDependencyByParentRepo(const std::string& dependsId,
                                              const std::shared_ptr<IAddon>& parent,
+                                             const std::string& parentRepoId,
```

Comment:
Is the `const std::shared_ptr<IAddon>& parent,` then still needed for something in future? Or can it be replaced by the new, as it is unused in function?

I think to have would be good as also unsued, as the function search about this addon. Only not 100% sure.

Reply:
it‘s only used for the debug-log-entry `dependee`. agree it should be removed then